odl/space/cu_ntuples.py

```diff
@@ -847,36 +976,271 @@ def _norm_default(x):
     return x.data.norm()
 
 
+def _pnorm_default(x, p):
+    if p == float('inf'):
+        raise NotImplementedError('inf-norm not implemented.')
+    elif int(p) != p:
+        raise NotImplementedError('non-integer powers not implemented')
+    # TODO: optimized version in C++ code?
```

Comment:
Its quite simple, I could do it for you. Simply give me the interface and I'll do it. (unless you want to learn how `odlpp` works ofc).

I clearly should have meant pulled, not commited :D